documentation/docs/30-advanced/10-advanced-routing.md

```diff
@@ -227,9 +225,41 @@ src/routes/
 └ +layout.svelte
 ```
 
+#### Breaking out of the root layout
+
+There is no way to break out of the root layout. You can be sure that it's always present in your app and for example put app-wide UI or behavior in it - and if you don't define one, it falls back to a simple `<slot />` component. But what to do, if you want some pages to _not_ have the same layout as the rest? You have two options:
```

Comment:
```suggestion
There is no way to break out of the root layout. You can be sure that it's always present in your app and for example put app-wide UI or behavior in it - and if you don't define one, it falls back to a simple `<slot />` component. If you want some pages to _not_ have the same layout as the rest, then you have two options:
```

Reply:
I think this is probably a good suggestion. Though I might make it "But if" instead of just "If"